### PR TITLE
Linked What's New entries to regular docs, maintenance/update pass

### PR DESCRIPTION
**Pull request by mikehoffms**
**Rendered:**
https://review.docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/whats-new/whats-new?branch=pr-en-us-1881

Updated all Experimental Features, in a maintainable way.  Used full-text searches:
```
![Experimental feature
Update: This feature has been released
```

### CODE REVIEW

**Comment by vchapel** [comment]
Some files, such as What's New in DevTools (Microsoft Edge 89) include the Experimental image and a preceding announcement that the feature is no longer experimental.

```
Update: This feature has been released and is no longer experimental.  See [Welcome tool](../../../welcome/welcome-tool.md).
![Experimental feature.](../../media/2020/06/experimental-tag-14px.msft.png)
```

This is confusing to me.

**Review by vchapel — APPROVED** [review]
LGTM - left one comment, not worth delaying these updates.

**Comment by mikehoffms** [comment]
> Some files, such as What's New in DevTools (Microsoft Edge 89) include the Experimental image and a preceding announcement that the feature is no longer experimental.

Resolved; normalized; clean presentation on all items that were or are marked Experimental.
@vchapel

File: microsoft-edge/devtools-guide-chromium/whats-new/includes/edge-whats-new-note.md [hunk 1]
```diff
@@ -3,6 +3,10 @@ author: MSEdgeTeam
 ms.author: msedgedevrel
 ms.prod: microsoft-edge
 ms.topic: include
-ms.date: 02/12/2021
+ms.date: 05/13/2021
```
Comment (mikehoffms) [inline microsoft-edge/devtools-guide-chromium/whats-new/includes/edge-whats-new-note.md:6]: @zoherghadyali  - fyi in the next What's New, I'll ask you to review this wording for top of every What's New.

**Pull request merged**